Sortng.py.txt

```diff
@@ -0,0 +1,108 @@
+import time
+#1
+lista=[1,2,5,6,89,10,22,1,30,100,2]
+start=time.clock()
+print("Oryginalna lista",lista)
+wp=int(input('Podaj wartość progową: '))
+tab=[]
+m=0
+for i in range(0,len(lista)):
+    if lista[m]>wp:     
+        tab.append(lista[m])
+        m+=1
+    else:
+        m+=1
+print("wszystkie liczby przekraczające wartość progową: ",tab)
+if len(tab)>=3:
+    for j in range(0,len(tab)):
+        m=0
+        for j in range(0,(len(tab)-1)):
+            if tab[m]>tab[m+1]:
+                z=tab[m]
+                tab[m]=tab[m+1]
+                tab[m+1]=z
+                m=m+1
+            else:
+                m=m+1
+
+    
+    print("trzy pierwsze wartości, które przekraczają wartość progową- od największej do najmniejszej:",tab[2],tab[1],tab[0])
+    for i in range(0,len(lista)):
+        r=0
+        for i in range(0,(len(lista)-1)):
+            if lista[r]==tab[0]:
+                r
+            else:
+                r+=1
+        p=0
+        for i in range(0,(len(lista)-1)):
+            if lista[p]==tab[1]:
+                p
+            else:
+                p+=1
+        s=0
+        for i in range(0,(len(lista)-1)):
+            if lista[s]==tab[2]:
+                s
+            else:
+                s+=1
+    end= time.clock()
+    print("znajdują się pod indeksami: ",s,p,r)
+else:
+    end= time.clock()
+    print("za mało licz przekracza próg!!!")
+total=end-start
+print("{0:02f}s".format(total))
+
+#2
```

Comment:
znowu komentarz #2, który nic nie mówi. Ja nie wiem czy to zadanie 2 czy to algorytm drugi(?) nie musisz odpowiadać tutaj, nawet tego nie chcę. Chcę żebyś pisała kod tak, żeby nie trzeba było komentarzy, kod sam w sobie po nazwach funkcji, zmiennych, strukturze ma być czytelny. Przeczytaj początek książki clean code.